webapp/src/js/enketo/medic-xpath-extensions.js

```diff
@@ -167,18 +203,12 @@ module.exports = {
     },
     'to-bikram-sambat': convertToBikramSambat,
     'parse-timestamp-to-date': parseTimestampToDate, // Function name convention of XForm
-    'difference-in-months': function(d1, d2) {
-      const d1Moment = asMoment(d1);
-      const d2Moment = asMoment(d2);
-
-      if (!d1Moment.isValid() || !d2Moment.isValid()) {
-        return XPR.string('');
-      }
-
-      const months = d2Moment.diff(d1Moment, 'months');
-      return XPR.number(months);
-    },
-    'cht:extension-lib': function() {
+    'difference-in-days': (d1, d2) => dateDiff(d1, d2, TIME_UNIT.DAYS),
+    'days-remaining-in-week': (d1, d2) => dateDiff(d1, d2, TIME_UNIT.DAYS_REMAINING),
+    'difference-in-weeks': (d1, d2) => dateDiff(d1, d2, TIME_UNIT.WEEKS),
+    'difference-in-months': (d1, d2) => dateDiff(d1, d2, TIME_UNIT.MONTHS),
+    'difference-in-years': (d1, d2) => dateDiff(d1, d2, TIME_UNIT.YEARS),
```

Comment:
As I mentioned in the issue thread, I think we want to preface all these function names with `cht:` (so `cht:difference-in-days`, etc).  For `difference-in-months` lets just have two entries.  One for the deprecated `difference-in-months` and the new `cht:difference-in-months`.

Reply:
Prefixed function keys with `cht:` and left a comment next to the future deprecated `difference-in-months` function.